source/config.js

```diff
@@ -220,21 +226,42 @@ module.exports = argv.default(defaultConfig).default(rcConfig).argv;
 
 module.exports.homedir = homedir;
 
+let fileSeparator = module.exports.fileSeparator;
+
 let currentRootPath = module.exports.rootPath;
 if (typeof currentRootPath !== 'string') {
   currentRootPath = '';
 } else if (currentRootPath !== '') {
-  // must start with a slash
-  if (currentRootPath.charAt(0) !== '/') {
-    currentRootPath = '/' + currentRootPath;
+  // must start with a slash (on unix-like systems)
+  if (currentRootPath.charAt(0) !== fileSeparator && isWindows !== true) {
```

Comment:
This change is not mandatory, everything works fine without it. On the other hand, there's no point in adding root slash on windows.

BTW, as ungit requires node >= 4 this could be replaced by
```javascript
if (!path.isAbsolute(currentRootPath) { .. }
```
As far as I know prior node 4 `path.isAbsolute()` was not always returning boolean (behaved differently on win and unix).